auth/src/main/java/io/grpc/auth/GoogleAuthLibraryCallCredentials.java

```diff
@@ -56,6 +58,8 @@
  * Wraps {@link Credentials} as a {@link CallCredentials}.
  */
 final class GoogleAuthLibraryCallCredentials implements CallCredentials {
+  private static final String SERVICE_ACCOUNT_CLASS_NAME
+      = "com.google.auth.oauth2.ServiceAccountCredentials";
```

Comment:
I can't do that, because that would require `ServiceAccountCredentials.class` to be available. I'm trying to make it so that if `ServiceAccountCredentials` isn't on the classpath, the code will still work. Doing `instanceof ServiceAccountCredentials` and `ServiceAccountCredentials.class` both fail if the class is not available.